argocd-managed/prometheus/values.yaml

```
@@ -0,0 +1,5 @@
+grafana:
+  admin:
+    existingSecret: "kibana-credentials"
+    userKey: KIBANA_USERNAME
+    passwordKey: KIBANA_PASSWORD
```

To place values for a subchart you must place them under an object with the same name ([or alias](https://helm.sh/docs/topics/charts/#alias-field-in-dependencies))
```suggestion
prometheus-stack:
  grafana:
    admin:
      existingSecret: "kibana-credentials"
      userKey: KIBANA_USERNAME
      passwordKey: KIBANA_PASSWORD

```